src/Components/Common/Header.css

```diff
@@ -0,0 +1,13 @@
+header {
+    padding: 20px;
+    justify-content:center; /* Distributes space between elements */
+    align-items: center; /* Aligns elements vertically */
+  }
```

Comment:
please change the `colour` of this header, it's too dark you can barely read it, and that's not good for accessibility and UX 👍